### PR TITLE
Set version to .dev for the master branch

### PR DESCRIPTION
**Pull request by goosemania**
A correct version would be shown for installations from the master branch.
Bindings will be generated with a proper version as well.

re #4936
https://pulp.plan.io/issues/4936

### CODE REVIEW

**Comment by goosemania** [comment]
I think 0.2.0b1 hasn't been released yet, so 0.2.0b1.dev should be the one. @bmbouter , @daviddavis ?

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 setup(
     name='pulp-ansible',
-    version='0.2.0b1',
+    version='0.2.0b1.dev',
```
Comment (bmbouter) [inline setup.py]: Also every plugin has the version in a second place too. It's the __version__ = in the __init__.py file of the package. That needs to be changed for any of them also.

Comment (goosemania) [reply setup.py]: Is it for towncrier purposes?

Comment (bmbouter) [reply setup.py]: Not especially for towncrier, although towncrier does use it. It's from pep396 https://www.python.org/dev/peps/pep-0396/. If we were doing it super to the spec we would derive the setup.py from __version__ or vice versa as in [step 8](https://www.python.org/dev/peps/pep-0396/#specification). I think using `bumpversion` would be good too.

**Comment by bmbouter** [comment]
I agree it should be 0.2.0b1.dev because we haven't released 0.2.0b1 yet.

**Comment by bmbouter** [comment]
Thanks @goosemania !

**Pull request merged**